CoverageProfiler/CoverageProfiler.inputs.json

```diff
@@ -0,0 +1,32 @@
+{
+  "coverageProfile.referenceFasta": "File",
+  "coverageProfile.SamtoolsDepth.samtools_docker": "String (optional, default = \"euformatics/samtools:1.20\")",
+  "coverageProfile.CovProfileViz.cpu": "Int? (optional)",
+  "coverageProfile.DepthOfCoverage.gatk_docker": "String (optional, default = \"broadinstitute/gatk:4.5.0.0\")",
+  "coverageProfile.referenceDict": "File",
+  "coverageProfile.intervals": "File",
+  "coverageProfile.referenceFai": "File",
+  "coverageProfile.alignedBam": "File",
+  "coverageProfile.alignedBamIndex": "File",
+  "coverageProfile.coverageTool": "String (optional, default = \"Samtools\")",
+  "coverageProfile.RegionalDepthProfile.mem_gb": "Int (optional, default = 7)",
+  "coverageProfile.RegionalDepthProfile.disk_size_gb": "Int? (optional, default = 500)",
+  "coverageProfile.DepthOfCoverage.cpu": "Int? (optional)",
+  "coverageProfile.SamtoolsDepth.cpu": "Int? (optional)",
+  "coverageProfile.MinMappingQuality": "Int (optional, default = 20)",
+  "coverageProfile.CovProfileViz.mem_gb": "Int (optional, default = 32)",
+  "coverageProfile.MinBaseQuality": "Int (optional, default = 20)",
+  "coverageProfile.cnv_depth_profile": "Boolean (optional, default = false)",
+  "coverageProfile.CovProfileViz.CovProfileViz_docker": "String (optional, default = \"us-central1-docker.pkg.dev/tag-team-160914/gptag-dockers/covprofileviz:0.0.0\")",
+  "coverageProfile.CovProfileViz.preemptible": "Int? (optional, default = 3)",
```

Comment:
I know it is common to use a default of 3, but I think it is usually best to do a default of 1.

Where is this default getting enforced?  Is this enforced by the WDL?

Reply:
It is enforced by the WDL.